api-guidelines/rest/http/headers/rules/must-forward-test-header.md

```diff
@@ -0,0 +1,9 @@
+---
+id: R000081
+---
+
+# MUST forward `Test` header
+
+An API provider or consumer that makes API requests as a direct consequence of an API request or response flagged with a test header must add the [`Test` header](./may-use-header.md) of the triggering interaction to all following requests and subsequent asynchronous events.
```

Comment:
Phew, this is a very long sentence and hard to understand. Can we shorten it, e.g. like this?

An API provider or consumer must include the [`Test` header](./may-use-header.md) from the initial interaction in all subsequent requests and asynchronous events if the original API request or response is marked with a Test header.

Reply:
BUT: again, I thought this is about synchronous calls? Why are you mentioning asynchronous events here?

Reply:
see above. I will shorten the sentence.